core/crypto/whitebox/aes_whitebox.cpp-chow-sample

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 AES WBC Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
The `LICENSE` file is missing. This should ideally be somewhere in `thirdparty` to be properly identified as non-Godot code.